src/features/island/buildings/components/building/workBench/components/ToolsGuide.tsx

```
@@ -158,6 +161,21 @@ export const ToolsGuide: React.FC = () => {
             setShowBoostsKey={setShowBoostsKey}
           />
         ))}
+        <div className="w-full pt-2 pb-0">
+          <Label type="default">
+            {t("waterTools")}
+          </Label>
```

ESLint

[error] Replace `⏎············{t("waterTools")}⏎··········` with `{t("waterTools")}`

(prettier/prettier)